### PR TITLE
Support encrypted columns in Schema Loader

### PR DESCRIPTION
**Pull request by brfrn169**
## Description

This PR adds support for encrypted columns to Schema Loader. With this change, users can specify the `ENCRYPTED` flag in the column definition as follows:

```json
{
  "sample_db.sample_table": {
    "partition-key": [
      "c1"
    ],
    "clustering-key": [
      "c4 ASC",
      "c6 DESC"
    ],
    "columns": {
      "c1": "INT",
      "c2": "TEXT ENCRYPTED",
      "c3": "BLOB",
      "c4": "INT",
      "c5": "BOOLEAN ENCRYPTED",
      "c6": "INT"
    }
  }
}
```

In the above example, the `ENCRYPTED` flag is specified for the `c2` and `c5` columns:

## Related issues and/or PRs

- https://github.com/scalar-labs/scalardb/pull/1907

## Changes made

- Added support for encrypted columns to Schema Loader.

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

Added support for encrypted columns to Schema Loader.

### CODE REVIEW

File: core/src/main/java/com/scalar/db/api/TableMetadata.java [hunk 1]
```diff
@@ -162,7 +163,8 @@ public boolean equals(Object o) {
         && Objects.equals(partitionKeyNames, metadata.partitionKeyNames)
         && Objects.equals(clusteringKeyNames, metadata.clusteringKeyNames)
         && Objects.equals(clusteringOrders, metadata.clusteringOrders)
-        && Objects.equals(secondaryIndexNames, metadata.secondaryIndexNames);
+        && Objects.equals(secondaryIndexNames, metadata.secondaryIndexNames)
+        && Objects.equals(encryptedColumnNames, metadata.encryptedColumnNames);
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/api/TableMetadata.java:167]: I forgot to add `encryptedColumnNames` to `equals()` and `hashCode()` in the previous PR...

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**